.github/workflows/test-with-pypi.yml

```diff
@@ -0,0 +1,71 @@
+name: Test with PyPI
+
+on: [pull_request, workflow_dispatch]
+
+env:
+  PYTHONUNBUFFERED: 1
+
+jobs:
+
+  # Test against PyPI packages
+  test-with-pypi-ubuntu:
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']
+        toolkit: ['pyside6']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
@mdickinson any preference on which versions to test? Do really want all or should we use 3.8 and 3.11 only?

Reply:
Dropping 3.7 from the workflow would be reasonable if we're also dropping 3.7 support for Chaco in general. If that's what we want to do then that seems fine to me - 3.7 EOL is June this year.

Reply:
@dpinte Ah, sorry; I did miss that there was a second job below that also goes through all combinations. As a compromise, I'd suggest testing on all 5 versions on Ubuntu, but limiting ourselves to 3.8 and 3.11 on Windows and macOS.